### PR TITLE
Update the Jakarata security FAT tests to use the default RS256 

### PR DESCRIPTION
**Pull request by c00crane**
During test development, it was easier to use HS256 for the Jakarta 10 security tests.
I'm updating the tests to use the default.
Another pulll will deliver tests targeting the configuration of signature algorithms.

### CODE REVIEW

File: dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/token/TokenResponseValidator.java [hunk 1]
```diff
@@ -10,138 +10,146 @@
...
+        }
+        return issuerconfigured;
+    }
+
+    JwtClaims validateIdTokenFormat(String idtoken, JwtContext jwtcontext, String clientSecret, String issuerconfigured) throws Exception {
+        JsonWebStructure jws = JwtParsingUtils.getJsonWebStructureFromJwtContext(jwtcontext);
+        Key verificationKey = JwtUtils.getJwsVerificationKey(jws, clientConfig);
+
+        JwtConsumerBuilder builder = new JwtConsumerBuilder();
+        builder.setJwsAlgorithmConstraints(new AlgorithmConstraints(ConstraintType.WHITELIST, MetadataUtils.getIdTokenSigningAlgorithmsSupported(clientConfig)));
```
Comment (tevans78) [inline dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/token/TokenResponseValidator.java:140]: While I understand that you probably can not change "WHITELIST" straight away, please open an issue or PR on the upstream library to get it changed, e.g. "ALLOWLIST"

**Review by helyarp — COMMENTED** [review]
Looks good!

File: dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/token/TokenResponseValidator.java (hunk 1 above)
Comment (c00crane) [reply dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/token/TokenResponseValidator.java:140]: @ayoho

Comment (ayoho) [reply dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/token/TokenResponseValidator.java:140]: It is actually already updated in a newer version of the jose4j library, we just haven't updated our usage of the library yet.

**Comment by c00crane** [comment]
#libby
#build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,
io.openliberty.security.jakartasec.3.0.internal_fat.config,
io.openliberty.security.jakartasec.3.0.internal_fat.logout,
io.openliberty.security.jakartasec.3.0.internal_fat.refresh

**Review by ayoho — APPROVED** [review]

**Comment by ayoho** [comment]
#libby

**Comment by c00crane** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_fFQqJmpyEe2VEtMj7GKy_A

**Comment by c00crane** [comment]
#libby-build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,
io.openliberty.security.jakartasec.3.0.internal_fat.config,
io.openliberty.security.jakartasec.3.0.internal_fat.logout,
io.openliberty.security.jakartasec.3.0.internal_fat.refresh

**Pull request merged**